integration/go.mod

```diff
@@ -363,15 +363,3 @@ require (
 replace github.com/onflow/flow-go => ../
 
 replace github.com/onflow/flow-go/insecure => ../insecure
-
-// the replaces below should not be needed once flow-go and all the repos are updated
-// to use Cadence 1.0
-replace github.com/onflow/cadence => github.com/onflow/cadence v0.42.8
-
-replace github.com/onflow/flow-ft/lib/go/contracts => github.com/onflow/flow-ft/lib/go/contracts v0.7.1-0.20230711213910-baad011d2b13
-
-replace github.com/onflow/flow-nft/lib/go/contracts => github.com/onflow/flow-nft/lib/go/contracts v1.1.0
-
-replace github.com/onflow/sdks => github.com/onflow/sdks v0.5.0
-
-replace github.com/onflow/flow-core-contracts/lib/go/contracts => github.com/onflow/flow-core-contracts/lib/go/contracts v0.15.1-0.20231219201108-fbdb10b0a2da
```

Comment:
Removing this `replace` seems problematic. Without the `replace`, we're using `v1.2.4-0.20231016154253-a00dbf7c061f`, which is [this commit](https://github.com/onflow/flow-core-contracts/commit/a00dbf7c061f) from Oct 16 2023, on a feature branch. So it wouldn't include https://github.com/onflow/flow-core-contracts/pull/397.

Removing all the other `replace`s looks fine as-is.

Reply:
@tarakby This module imports a version of `flow-emulator` on a branch: [tarak/v0.58.1-0-with-crypto-type-fixes](https://github.com/onflow/flow-emulator/compare/tarak/v0.58.1-0-with-crypto-type-fixes). That branch is still importing the old `flow-core-contracts` [version](https://github.com/onflow/flow-emulator/blob/d334a0fcd3800857221dde01d6b51e20cafe8cd3/go.mod#L18), whereas `master` has been updated to use a newer [version](https://github.com/onflow/flow-emulator/blob/master/go.mod#L18). 

The older `flow-core-contracts` version being referenced on the `flow-emulator` branch is before the out-of-order version `v1.2.3` was retracted, so Go still tags it as `v1.2.4...` and considers it newer than everything else. That seems to be the behaviour anyway. 

I think updating the version of `flow-core-contracts` on your `flow-emulator` branch will allow us to also remove this last `replace` and use the right `core-contracts` version.